ontology/yaml/resources/HVAC/entity_types/DOAS.yaml

```diff
@@ -16,6 +16,22 @@
 ### Canonical Types  ###
 ########################
 
+DOAS_HTWHLSTC_SFVSC_EFVSC_EDM_SDM_RFC_SFC_FDPM2X_RTM:
+  guid: "e0627d5a-99cd-4edd-824b-636428427980"
+  description: "DOAS with supply and return fans, heatwheel and fan speed flowrate control"
+  is_canonical: temperature
+  implements:
+  - DOAS
+  - HTWHLSTC
+  - SFVSC
+  - EFVSC
+  - EDM
+  - SDM
+  - RFC
+  - SFC
+  - FDPM2X
```

Comment:
Do you have any filter alarms on the device?

Reply:
No filter alarms for this just the filter differential pressure sensors. Thanks